docs/Using Fleet/manage-access.md

```diff
@@ -111,13 +113,14 @@ Users with global access have access to all
 [hosts](https://fleetdm.com/docs/using-fleet/rest-api#hosts), [software](https://fleetdm.com/docs/using-fleet/rest-api#software), [queries](https://fleetdm.com/docs/using-fleet/rest-api#queries), and [policies](https://fleetdm.com/docs/using-fleet/rest-api#policies). Check out [the user permissions
 table](#user-permissions) above for global user permissions.
 
-Users can be assigned to multiple teams in Fleet.
+Users can be a member of multiple teams in Fleet.
```

Comment:
Let's revert to the old language:

```suggestion
Users can be assigned to multiple teams in Fleet.
```

(We recently updated the UI to change the "team members" terminology to reference users instead, and updated  the docs to match.)